controls/stig_rhel8.yml

```diff
@@ -1140,9 +1140,7 @@ controls:
         levels:
             - medium
         title: RHEL 8 passwords for new users must have a minimum of 15 characters.
-        rules:
-            - accounts_password_minlen_login_defs
-        status: automated
+        status: inherently met
```

Comment:
Wait, RHEL8 doesn't have a default minimum password length of 15 characters...

Reply:
I think this control is also enforced by the rule accounts_password_pam_minlen, The variable used in this rule has default value of 15. So do you think the status should be changed back to automated?

Reply:
If rule `accounts_password_pam_minlen` needs to be applied then I think this should be set back to `automated`.